.infrastructure/secret.yml

```diff
@@ -1,8 +1,13 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: app-secret
+  name: todoapp-secret
   namespace: todoapp
 type: Opaque
 data:
-  SECRET_KEY: QGUyKHl4KXYmdGdoM19zPTB5amEtaSFkcGVieHN6XmRnNDd4KS1rJmtxXzN6Zio5ZSoK
\ No newline at end of file
+  SECRET_KEY: QGUyKHl4KXYmdGdoM19zPTB5amEtaSFkcGVieHN6XmRnNDd4KS1rJmtxXzN6Zio5ZSoK
+  NAME: bXlkYXRhYmFzZQ==
+  USER: bXl1c2Vy
+  PASSWORD: bXlwYXNzd29yZA==
+  HOST: bXlzcWwtMC5teXNxbC1oLmRlZmF1bHQuc3ZjLmNsdXN0ZXIubG9jYWw=
```

Comment:
Wrong host `mysql-0.mysql-h.default.svc.cluster.local`. Take a look on the namespace name.